usr/toolchain-nxdk.cmake

```diff
@@ -0,0 +1,29 @@
+set(NXDK_DIR $ENV{NXDK_DIR})
+
+set(CMAKE_SYSTEM_NAME Generic)
+set(TOOLCHAIN_PREFIX nxdk)
+
+set(WIN32 1)
+
+#FIXME: Include stuff for MinGW / Windows-GNU / Windows-Clang?
+#FIXME: How to set the ABI?
+
+set(NXDK_LINK_XBE "${NXDK_DIR}/usr/bin/${TOOLCHAIN_PREFIX}-exe2xbe -OUT:<TARGET>.xbe <TARGET>.exe")
+
+# cross compiler to use for C
+set(CMAKE_C_COMPILER "${NXDK_DIR}/usr/bin/${TOOLCHAIN_PREFIX}-cc")
+set(CMAKE_C_STANDARD_LIBRARIES "${NXDK_DIR}/lib/libwinapi.lib ${NXDK_DIR}/lib/xboxkrnl/libxboxkrnl.lib ${NXDK_DIR}/lib/libxboxrt.lib  ${NXDK_DIR}/lib/libpdclib.lib ${NXDK_DIR}/lib/libnxdk_hal.lib ${NXDK_DIR}/lib/libnxdk.lib ${NXDK_DIR}/lib/libnxdk_usb.lib") #"${CMAKE_CXX_STANDARD_LIBRARIES_INIT}"
```

Comment:
If we do binary wrappers we should also fix the nxdk Makefile to benefit from that to avoid redundancy of CFLAGS / CXXFLAGS and friends.
We'll probably have to list the standard libraries redundant anyway, unless we automatically add them in the nxdk-ld wrapper.